docs/admin_guide/user-administration.rst

```diff
@@ -80,3 +80,51 @@ If a user runs out of projects and wants to create more, they should contact an
    :figclass: align-center
 
    Increase number of projects 
+
+
+Reset users' password
+---------------------
+
+If a user forgets his/her password and security quesion, an administrator can reset it by going to the edit user page as shown above and selecting reset user password button.
+
+.. _user-admin5.png: ../_images/admin/user-admin5.png
+.. figure:: ../imgs/admin/user-admin5.png
+   :alt: Reset password 
+   :target: `user-admin5.png`_
+   :align: center
+   :figclass: align-center
+
+   Reset password
+
+Before confirming the reset action make sure the user email shown in the confirm dialog box is the user you want to reset the password for.
+
+.. _user-admin6.png: ../_images/admin/user-admin6.png
+.. figure:: ../imgs/admin/user-admin6.png
+   :alt: Confirm user email 
+   :target: `user-admin6.png`_
+   :align: center
+   :figclass: align-center
+
+   Confirm user email
+
+When sure that you have the right user click yes.
+
+.. _user-admin7.png: ../_images/admin/user-admin7.png
+.. figure:: ../imgs/admin/user-admin7.png
+   :alt: Confirm password reset  
+   :target: `user-admin7.png`_
+   :align: center
+   :figclass: align-center
+
+   Confirm password reset
+
+The new autogenerated password will be shown. Make sure to copy it efore closing the dialog box.
```

Comment:
@ErmiasG `efore`. Also we can say that they need to copy it because it cannot be retrieved later.